spec/lib/pact_broker/api/decorators/labels_decorator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "pact_broker/api/decorators/labels_decorator"
+require "pact_broker/labels/service"
+
+module PactBroker
+  module Api
+    module Decorators
+
+      describe LabelsDecorator do
+        before do
+          td.create_consumer("Foo")
+            .create_label("consumer")
+            .create_consumer("Bar")
+            .create_label("provider")
+            .create_consumer("Wiffle")
+            .create_label("provider")
+        end
+
+        let(:label) do
+          PactBroker::Labels::Service.get_all_unique_labels
+        end
+
+        let(:options) { { user_options: { resource_url: "http://example.org/labels", hide_label_decorator_links: true } } }
```

Comment:
naming wise, this could be called differently, something like `hide_pacticipant_links`